eclipse-collections-code-generator/src/main/resources/test/list/immutable/immutablePrimitiveArrayListTest.stg

```diff
@@ -95,6 +99,20 @@ public class Immutable<name>ArrayListTest extends AbstractImmutable<name>ListTes
         Assert.assertEquals(4, list.binarySearch(<(literal.(type))("9")>));
         Assert.assertEquals(-6, list.binarySearch(<(literal.(type))("10")>));
     }
+<if(primitive.specializedStream)>
+
+    @Test
+    public void stream()
+    {
+        Assert.assertEquals(Arrays.asList(<["1", "2", "3"]:(literal.(type))(); separator=", ">), Immutable<name>ArrayList.newListWith(<["1", "2", "3"]:(literal.(type))(); separator=", ">).primitveStream().boxed().collect(Collectors.toList()));
+    }
+
+    @Test
+    public void parallelStream()
+    {
+        Assert.assertEquals(Arrays.asList(<["1", "2", "3"]:(literal.(type))(); separator=", ">), Immutable<name>ArrayList.newListWith(<["1", "2", "3"]:(literal.(type))(); separator=", ">).primitveParallelStream().boxed().collect(Collectors.toList()));
```

Comment:
Did you check code coverage? I'm curious for these small lists if trySplit() is getting called at all.

Reply:
Is is possible to have these assertions in the superclass, AbstractImmutable<name>ListTestCase?

Reply:
> coverage

Hi @motlin , How do you measure coverage? I tried to work with intellij and it does not give me coverage indicationi. by the way which ide do you use for developing this project. The stg files are not nice to edit,

Reply:
I use IntelliJ ultimate. But even with IntelliJ community you can do some lower-tech stuff to make sure that important parts are covered. In this method I'd just put breakpoints in trySplit() on every line I care about, run relevant tests in debug mode, remove all breakpoints that get hit, and treat any remaining breakpoints as uncovered code that needs new tests.

Reply:
I checked with debugger and it makes trysplit